app/(backend)/factories/serviceFactory.ts

```
@@ -0,0 +1,83 @@
+    if (!this.modelService) {
+      const modelRepo = new PrismaAIModelRepository();
+      this.modelService = new ModelService(modelRepo);
+    }
+    return this.modelService;
+  }
+  
+  /**
+   * Get ProviderService instance (singleton)
+   */
+  static getProviderService(): IProviderService {
+    if (!this.providerService) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)